docs/bigquery/snippets.py

```diff
@@ -440,64 +457,167 @@ def test_table_exists(client, to_delete):
+    table = client.create_table(table)
+    to_delete.insert(0, table)
+
+    # [START bigquery_update_table_description]
+    import datetime
+    import pytz
+
+    # client = bigquery.Client()
+    # table_ref = client.dataset('my_dataset').table('my_table')
+    # table = client.get_table(table_ref)  # API request
+
+    assert table.expires is None
+
+    # set table to expire 5 days from now
+    expiration = datetime.datetime.now(pytz.utc) + datetime.timedelta(days=5)
+    table.expires = expiration
+    table = client.update_table(table, ['expires'])  # API request
+
+    # expiration is stored in milliseconds
+    margin = datetime.timedelta(microseconds=1000)
+    assert expiration - margin <= table.expires <= expiration + margin
+    # [END bigquery_update_table_description]
+
+
+def test_add_empty_column(client, to_delete):
+    """Adds an empty column to an existing table."""
+    dataset_id = 'add_empty_column_dataset_{}'.format(_millis())
+    table_id = 'add_empty_column_table_{}'.format(_millis())
+    dataset = bigquery.Dataset(client.dataset(dataset_id))
+    dataset = client.create_dataset(dataset)
+    to_delete.append(dataset)
+
+    table = bigquery.Table(dataset.table(table_id), schema=SCHEMA)
+    table = client.create_table(table)
+    to_delete.insert(0, table)
+
+    # [START bigquery_add_empty_column]
+    # client = bigquery.Client()
+    # dataset_id = 'my_dataset'
+    # table_id = 'my_table'
+
+    table_ref = client.dataset(dataset_id).table(table_id)
+    table = client.get_table(table_ref)  # API request
+
+    original_schema = table.schema
+    new_schema = original_schema[:]  # creates a copy of the schema
+    new_schema.append(bigquery.SchemaField('phone', 'STRING'))
+
+    table.schema = new_schema
+    table = client.update_table(table, ['schema'])  # API request
+
+    assert len(table.schema) == len(original_schema) + 1 == len(new_schema)
+    # [END bigquery_add_empty_column]
+
+
+def test_relax_column(client, to_delete):
+    """Updates a schema field from required to nullable."""
+    dataset_id = 'relax_column_dataset_{}'.format(_millis())
+    table_id = 'relax_column_table_{}'.format(_millis())
+    dataset = bigquery.Dataset(client.dataset(dataset_id))
+    dataset = client.create_dataset(dataset)
+    to_delete.append(dataset)
+
+    # [START bigquery_relax_column]
+    # client = bigquery.Client()
+    # dataset_id = 'my_dataset'
+    # table_id = 'my_table'
+    original_schema = [
+        bigquery.SchemaField('full_name', 'STRING', mode='REQUIRED'),
+        bigquery.SchemaField('age', 'INTEGER', mode='REQUIRED'),
+    ]
+    table_ref = client.dataset(dataset_id).table(table_id)
+    table = bigquery.Table(table_ref, schema=original_schema)
+    table = client.create_table(table)
+    assert all(field.mode == 'REQUIRED' for field in table.schema)
+
+    # SchemaField properties cannot be edited after initialization.
+    # To make a changes, new SchemaField objects must be constructed.
```

Comment:
Nit: use active voice, not passive voice for instructions.

"To make changes, construct new SchemaField objects."